R/MsBackendMzR-functions.R

```diff
@@ -27,6 +27,8 @@ MsBackendMzR <- function() {
     msd <- mzR::openMSfile(x)
     on.exit(mzR::close(msd))
     hdr <- mzR::header(msd)
+    if (!is.data.frame(hdr))
```

Comment:
Quick question - when would `hdr` not be a `data.frame`? Shouldn't this rather be guaranteed by `mzR::header()`?

Reply:
good question - actually it's not. The *original* `mzR` code returns a `list` if the header of only one spectrum is extracted and a `data.frame` otherwise. We could this change in `mzR` (since it's also *cleaner*), but I'm afraid that might break code in other packages?

Reply:
I reverted that again. `mzR` did return before already a `data.frame` (otherwise this would have never worked before) - although documentation did tell something different. Based on the documentation I changed then the code in `mzR` in a recent PR (which never got into mzR *master* so far), which I did now again revert. So, to summarize, `mzR::header` did always return a `data.frame` and will return always a `data.frame`.